.changeset/hip-hotels-divide.md

```diff
@@ -0,0 +1,7 @@
+---
+"astro": minor
+---
+
+Remove content collection warning when a configured collection does not have a matching directory name. This should resolve `i18n` collection warnings for Starlight users.
```

Comment:
Micro nit to match our changeset style as per https://contribute.docs.astro.build/guides/new-feature-docs/#changesets

```suggestion
Removes content collection warning when a configured collection does not have a matching directory name. This should resolve `i18n` collection warnings for Starlight users.
```